routes/web.php

```diff
@@ -36,4 +38,16 @@
 Route::post('editUser', [RegisterController::class, 'update']);
 Route::get('search-users', [RegisterController::class, 'search']);
 
+Route::get('/parents', [ParentController::class, 'index'])->name('parents.index');
+Route::post('/parents', [ParentController::class, 'store'])->name('parents.store');
+Route::resource('parents', ParentController::class);
+Route::get('parents/search', [ParentController::class, 'search'])->name('parents.search');
+Route::get('/parents/{id}/edit', [ParentController::class, 'edit'])->name('parents.edit');
```

Comment:
wil je de id van de parent echt laten zien in de url? het is info die de gebruiker nooit hoeft te zien